corehq/apps/user_importer/importer.py

```diff
@@ -306,7 +306,7 @@ def create_or_update_web_user_invite(email, domain, role_qualified_id, upload_us
     if invite_created and send_email:
         invite.send_activation_email()
     if invite_created and user_change_logger:
-        user_change_logger.add_change_message(_("Invited to domain"))
+        user_change_logger.add_change_message(_("Invited to domain {domain_name}").format(domain_name=domain))
```

Comment:
nit: quote domain name
```suggestion
        user_change_logger.add_change_message(_("Invited to domain '{domain_name}'").format(domain_name=domain))
```

Reply:
https://github.com/dimagi/commcare-hq/pull/29935/commits/15e180758f65e1ac9a952e36fc8de8bedf73fa46